.buildkite/scripts/steps/functional/uptime_plugin.sh

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+source .buildkite/scripts/common/util.sh
+
+.buildkite/scripts/bootstrap.sh
+.buildkite/scripts/download_build_artifacts.sh
+
+export JOB=kibana-uptime-plugin
+
+echo "--- Uptime plugin @elastic/synthetics Tests"
+
+cd "$XPACK_DIR"
+
+node plugins/synthetics/scripts/uptime_e2e.js --kibana-install-dir "$KIBANA_BUILD_LOCATION" ${GREP:+--grep \"${GREP}\"}
```

Comment:
What's the grep portion used for?

Reply:
it's primarily used to filter out tests by journey name